src/sds.c

```diff
@@ -476,6 +479,20 @@ sds sdscatlen(sds s, const void *t, size_t len) {
     return s;
 }
 
+/* This method copies the sds `s` into `buf` which is the target character buffer. */
+size_t sdscopytobuffer(unsigned char *buf, size_t buf_len, const sds s, uint8_t *hdr_size) {
+    /* min amount of bytes required to store the sds header + data + NULL */
+    size_t sdsminlen = sdslen(s) + sdsHdrSize(s[-1]) + 1;
+    size_t required_keylen = sdsminlen;
```

Comment:
```suggestion
    size_t required_keylen = sdslen(s) + sdsHdrSize(s[-1]) + 1;
```
it seems we don't use `sdsminlen` again, required_keylen seems not good, how about `required_len`